File: csharp/fizzbuzz.cs
class FizzBuzz 
{
	const string FIZZ = "Fizz";
	const string BUZZ = "Buzz";
	const int STOP = 100;
	static void Main() 
	{
		for (int i = 1; i <= STOP; i++) {
			if (i % 15 == 0) {
				System.Console.WriteLine(FIZZ + BUZZ);
			} else if (i % 3 == 0) {
				System.Console.WriteLine(FIZZ);
			} else if (i % 5 == 0) {
				System.Console.WriteLine(BUZZ);
			} else {
				System.Console.WriteLine(i);
			}
		}
	}
}

File: scala/FizzBuzz.scala
object FizzBuzz {
	val FIZZ = "Fizz";
	val BUZZ = "Buzz";
	val STOP = 100;
	def main(args: Array[String]): Unit = {
		for (i <- 1 to STOP) {
			if (i % 15 == 0) {
				println(FIZZ + BUZZ);
			} else if (i % 3 == 0) {
				println(FIZZ);
			} else if (i % 5 == 0) {
				println(BUZZ);
			} else {
				println(i);
			}
		}
	}
}
